docs/apis/migration-api-overview.md

```diff
@@ -337,6 +337,51 @@ File content that is referenced within the manifest of the package structure mus
 
 The main requirement for the structure is that the FileValue references in the **Manifest.XML** file must refer to the exact name and physical hierarchy that the content is stored in within the Azure Blob Store location for import. The destination file names and folder hierarchy from the import operation aren't directly related to the physical naming and hierarchy and are instead defined through the **Manifest.XML** file.
 
+### ArchivedFiles.XML
+
+The **ArchivedFiles.XML** file, if included, is expected to be at the root of the Azure blob storage container defined by the CreateMigrationJob’s azureContainerManifestUri parameter. The QuickXorHash/Checksum has to be computed for the optional pack0.zip file using the [QuickXorHash Algorithm](/onedrive/developer/code-snippets/quickxorhash). If small files in the Manifest.xml are not archived (not showing in ArchiveFiles.xml), they still can be processed as before (equivalent to feature off).
+
+This file allows for transferring files in batch. When the ArchivedFiles.xml is provided by the client (the default scenario), the server side will validate the correctness of the .xml schema, then import the files inside each pack. When ArchivedFiles.XML is not provide by the client, the feature will be disabled. 
+
+#### Prerequisites
+
+The QuickXorHash/Checksum has to be computed for the optional pack0.zip file using the [QuickXorHash Algorithm](/onedrive/developer/code-snippets/quickxorhash).
+QuickXorHash is created for the .zip file which concatenates all smaller files.
+
+|Item|Requirement|
+|:-----|:-----|
+|Encryption|The zip file must be encrypted.|
+|File size|Less than 100KB. Note: File size is calculated based on the encrypted file.|
+|Compression|Do not compress the small data files your are archiving into the zip file.|
+|Zip file size|Maximum size 10MB|
+|Zip files per package|Maximum of 2|
+
+#### Table 1: Example ArchivedFiles.XML file
+
+```xml
+

```

Comment:
in the future, please don't include line breaks immediately after & before fenced code blocks